types/html-validator/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for html-validator 4.0
+// Project: https://github.com/zrrrzzt/html-validator
+// Definitions by: Takesi Tokugawa <https://github.com/TokugawaTakesi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+/**
+ * If desired validation format is 'json', really not json, JavaScript object (parsed object will be returned)
+ */
+declare function HtmlValidator(
+    options:
+        HtmlValidator.OptionsForHtmlFileAsValidationTargetAndObjectAsResults |
+        HtmlValidator.OptionsForExternalUrlAsValidationTargetAndObjectAsResults
+): Promise<HtmlValidator.ValidationResultsAsParsedJSON>;
+
+/**
+ * If desired validation format is not json, string will be returned;
+ */
+declare function HtmlValidator(
+    options:
+        HtmlValidator.OptionsForHtmlFileAsValidationTargetAndTextAsResults |
+        HtmlValidator.OptionsForExternalUrlAsValidationTargetAndTextAsResults
+): Promise<string>;
+
+declare namespace HtmlValidator {
+    interface BasicOptions {
+        validator?: Object;
```

Comment:
I do not know in advance which npm package will be use instead. Please give me the hint.